Tasks/ANTPreview/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -2,9 +2,9 @@
 // GENERATED FILE - DO NOT EDIT DIRECTLY
 // *******************************************************
 {
-  "loc.friendlyName": "Ant With CodeCoverage",
+  "loc.friendlyName": "Ant With Code Coverage",
```

Comment:
thanks for proactively fixing this

can you also make sure that all the labels in these tasks follow the camel casing